pages/[actionId].tsx

```diff
@@ -187,6 +188,15 @@ const ActionExecution: NextPage<ActionPageProps> = ({ actionData }) => {
             message={step.id}
           />
         );
+      case STEPS.select_and_review_proposal:
+        return (
```

Comment:
@RafeeqWeideman-YDS Please see my comment below about the naming convention of the steps and splitting this step into 2